### PR TITLE
Fix find psql libs

### PR DESCRIPTION
**Pull request by sosaavedra**
Hi Oliver,

Please review the changes.

Thanks and best regards,
Victor

### CODE REVIEW

**Comment by ogasser** [comment]
Please rebase against master to avoid including the other commits from `efficient_lookup` in this PR.

**Comment by sosaavedra** [comment]
Done, all commits not related to this fix were removed from the branch.

File: CMakeLists.txt [hunk 1]
```diff
@@ -301,6 +301,9 @@ ELSE (MYSQL_FOUND AND SUPPORT_MYSQL)
 ENDIF (MYSQL_FOUND AND SUPPORT_MYSQL)
 
 ### PostgreSQL
```
Comment (ogasser) [inline CMakeLists.txt]: Add one newline after `### PostgreSQL`

File: CMakeLists.txt [hunk 2]
```diff
@@ -301,6 +301,9 @@ ELSE (MYSQL_FOUND AND SUPPORT_MYSQL)
...
+### Bug in Ubuntu 14.04 does not find PSQL Libs
```
Comment (ogasser) [inline CMakeLists.txt]: Use single `#` when they are just comments and not a new section

**Comment by sosaavedra** [comment]
Changed it.

**Pull request merged**